backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/placement/PlacementService.java

```diff
@@ -6,15 +6,20 @@
 import java.util.List;
 
 @Service
-public class  PlacementService {
- 
-    private final PlacementRepo placementRepo ;
+public class PlacementService {
+
+    private final PlacementRepo placementRepo;
 
     public PlacementService(PlacementRepo placementRepo) {
         this.placementRepo = placementRepo;
     }
 
-    public List<Placement> getAllPlacements(){
-               return placementRepo.getAllPlacement();
+    public List<Placement> getAllPlacements() {
+        return placementRepo.getAllPlacement();
+    }
+
+    public Placement addNewPlacement(NewPlacementData requestPlacement) {
+        return placementRepo.addNewPlacement(requestPlacement);
     }
```

Comment:
implement the generateUuid() method and create a new placement object with id, and return it to the repo